client/src/Auth/authenticatedRoute.tsx

```diff
@@ -13,3 +13,11 @@ export function withAuthentication(matcher: Matcher<{}, Context>) {
         context.currentUser ? matcher : redirect('/auth'),
     );
 }
+
+export function withAdminContentProtection(matcher: Matcher<{}, Context>) {
+    return map((request: NaviRequest<Context>, context: Context) =>
+        context.currentUser && context.currentUser.is_admin
```

Comment:
why do we have `is_admin` and not `isAdmin`? Shouldn't everything be camelCased?